packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -104,11 +104,32 @@ class _DeepLinkListViewMainPanel extends StatelessWidget {
             return const _ValidatedDeepLinksView();
           case PagePhase.noLinks:
             // TODO(hangyujin): This is just a place holder to add UI.
-            return const Text('Your flutter project has no Links to verify.');
+            return const CenteredMessage(
+                'Your Flutter project has no Links to verify.');
+          case PagePhase.analyzeErrorPage:
+            assert(controller.currentAppLinkSettings?.error != null);
+            return Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                const Text(
+                  'Failed to retrieve deep links from the Flutter project. This can be a result of errors in the project.',
+                ),
+                const SizedBox(height: densePadding),
+                Expanded(
+                  child: OutlineDecoration(
+                    child: SingleChildScrollView(
+                        child: Text(
+                      controller.currentAppLinkSettings!.error!,
+                      style: Theme.of(context).errorTextStyle,
```

Comment:
put Theme.of(context) in a variable at the top of this build method since it is used multiple times, and then use the variable throughout this build method instead.